guides/06_FAQs.md

```diff
@@ -34,15 +34,38 @@ If you have feature requests or feedback, please fill out the [customer question
 
 The meetings created by the Amazon Chime Application and the SDK are distinct. Amazon Chime SDK is meant for customers who would like to build the experience into their existing applications and it is **NOT** a means to create an Amazon Chime meeting (with a PIN) that can be joined using our client application. You cannot use the Amazon Chime SDK to build a custom client to join an Amazon Chime meeting with a PIN.
 
+### Does the MeetingsNotificationsConfiguration only support SQS queues in us-east-1?
+
+Yes, we currently support SNS topics or SQS queues located in us-east-1 region only. 
+
+### How can I learn about interruptions to the Amazon Chime service?
+
+You can be notified about Amazon Chime service interruptions at the https://status.aws.amazon.com/ website. In addition you can set up [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) events via the [AWS Personal Health Dashboard](https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/) for the Amazon Chime service. You can find the Amazon Chime SLA on this [webpage](https://aws.amazon.com/chime/sla/).
+
 ## Browser support
 
 ### What browsers are supported by the Amazon Chime SDK for JavaScript?
 
 You can find the complete list of browsers in this link: [Supported Browsers](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-browsers). [WebRTC](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API) support in a browser is a prerequisite for Amazon Chime SDK to run. Browsers that do not offer WebRTC support, like Internet Explorer, are not supported.
 
-### How can I learn about interruptions to the Amazon Chime service?
+### I am unable to select audio output device in some browsers, is this a known issue?
 
-You can be notified about Amazon Chime service interruptions at the https://status.aws.amazon.com/ website. In addition you can set up [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) events via the [AWS Personal Health Dashboard](https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/) for the Amazon Chime service. You can find the Amazon Chime SLA on this [webpage](https://aws.amazon.com/chime/sla/).
+[Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1152401) and [Safari](https://bugs.webkit.org/show_bug.cgi?id=179415) have known issues disallowing them from listing audio output devices on these browsers. While clients can continue the meeting using the default device, they will not be able to select devices in meetings.
+
+### My local video disappear in Safari browsers, is this a known issue?
+
+Both desktop and iOS Safari have limitations when using a video camera so please make sure your video is not used
```

Comment:
I think you can explain why Safari has limtiations in the first sentence, e.g.

> macOS and iOS Safari browsers have limitations when you use a camera in other apps and tabs. For example, if you enable a video in two macOS Safari tabs, one video will go black. Make sure to close all other apps and tabs that are using the camera.

Reply:
Fixed.